src/Utils.ts

```diff
@@ -863,7 +863,7 @@ class DummyInputManagerCore {
     private _firstDummy: DummyInput | undefined;
     private _lastDummy: DummyInput | undefined;
     private _transformElements: HTMLElement[] = [];
-    private _scrollFrame: number | undefined;
+    private _scrollTimer: number | undefined;
```

Comment:
you should initialize timers to `0` because `clearTimeout` does not throw when called with a non-existing timer. That way we will have less unnecessary checks in the code, which add to bundle-size and complexity.

Reply:
That is an additional operation. And I never call clearTimeout without `if (timer)` first.

Reply:
And `if (timer)` is a much lighter check that unconditionally calling a function.

Reply:
Perhaps we're optimizing for the wrong thing here ?  there hasn't been any customer feedback about tabster performance (in Teams or otherwise), but there has been for bundle-size 🤷.

Reply:
`timer && clearTimeout(timer)` is not worse than `timer = 0; clearTimeout(timer)`.